src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTabController.js

```diff
@@ -0,0 +1,77 @@
+({
+    /**
```

Comment:
(Code has been moved into a helper in a pending feature branch. Review this in the feature branch)